package Exception.src;

public class AssertionError {

    public static void main(String[] args) {
        int a=1;
        assert (a>11);
    }
}
